Makefile

```diff
@@ -31,6 +37,17 @@ build: vendor build-internal
 build-internal:
 	$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
 
+$(MANPAGES): %: %.md
+	@sed -e 's/\((.*\.5.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' $<  | $(GOMD2MAN) -in /dev/stdin -out $@
```

Comment:
I can’t see that the two sed commands match anything in existing man pages. The skopeo reference is especially suspect.

Reply:
Yup cut & paste from skopeo Makefile (fixed)